Use standard system file locations.
Use distro's compilation options.
Prevent musl getcats localization segfault.

diff -Naur a/makefile b/makefile
--- a/makefile	2012-02-08 07:28:03.000000000 +0000
+++ b/makefile	2020-10-22 11:22:29.053732761 +0100
@@ -274,16 +274,12 @@
 #TRACING = -g -fno-omit-frame-pointer -fsanitize=undefined
 #TRACING = -g -fno-omit-frame-pointer -fsanitize=address -fsanitize-recover
 
-#PREFIX	= /home/jeff
-PREFIX	= /usr/local
+PREFIX	= /usr
 BINDIR	= $(PREFIX)/bin
-#LOUTLIBDIR	= $(PREFIX)/lout.lib
-#LOUTDOCDIR	= $(PREFIX)/lout.doc
-#MANDIR	= $(PREFIX)/lout.man
-LIBDIR	= $(PREFIX)/share/lout-$(VERSION)
-LOUTLIBDIR	= $(LIBDIR)/lib
-LOUTDOCDIR	= $(LIBDIR)/doc
-MANDIR	= $(LIBDIR)/man
+LIBDIR	= $(PREFIX)/share/lout
+LOUTLIBDIR = $(PREFIX)/share/lout
+LOUTDOCDIR = $(PREFIX)/share/doc/lout
+MANDIR = $(PREFIX)/share/man/man1
 
 LIBFONT = font
 LIBMAPS = maps
@@ -295,7 +291,7 @@
 CHARIN	= 1
 CHAROUT	= 0
 
-USELOC	= 1
+USELOC	= 0
 LOC_FR	= fr
 LOC_DE	= de
 
@@ -305,9 +301,6 @@
 ZLIB		=
 ZLIBPATH	=
 
-CC	= gcc
-#CC	= bgcc
-
 RCOPY	= cp -r
 
 MKDIR	= mkdir -p
@@ -315,7 +308,7 @@
 # Add WARN to COPTS for more checking
 WARN	= -Wpointer-arith -Wclobbered -Wempty-body -Wmissing-parameter-type -Wmissing-field-initializers -Wold-style-declaration -Wtype-limits -Wuninitialized -Winit-self -Wlogical-op -Wmissing-prototypes -Wmissing-declarations -Wnested-externs -Wbad-function-cast
 
-COPTS	= -ansi -pedantic -Wall -O3 -pipe
+COPTS	= -ansi -pedantic -Wall -pipe
 
 CFLAGS	= -DOS_UNIX=$(OSUNIX)					\
 	  -DOS_DOS=$(OSDOS)					\
@@ -349,7 +342,7 @@
 	  z49.o z50.o z51.o z52.o
 
 lout:	$(OBJS)
-	$(CC) $(CFLAGS) -o lout $(OBJS) $(ZLIB) -lm
+	$(CC) $(CFLAGS) $(LDFLAGS) -o lout $(OBJS) $(ZLIB) -lm
 	chmod a+x lout
 
 $(OBJS): externs.h
@@ -357,7 +350,7 @@
 externs.h:
 
 prg2lout:	prg2lout.c
-	$(CC) $(COPTS) -o prg2lout prg2lout.c
+	$(CC) $(COPTS) $(LDFLAGS) -o prg2lout prg2lout.c
 	chmod a+x prg2lout
 
 all:	lout prg2lout
